app/utils/constant.ts

```diff
@@ -1 +1,2 @@
-export const GRAMS_PER_TON = 1000000;
\ No newline at end of file
+export const GRAMS_PER_TON = 1000000;
+export const UINT256_DECIMALS = Math.pow(10, 6);
```

Comment:
this can be confusing, is it referring to ERC3525_VALUE_DECIMALS or just UINT256_DECIMALS ?

Reply:
Currently it's just used in a calldata with UINT256 parameters.
`calldata: tokens.length === 0 ? [amount * UINT256_DECIMALS, 0] : [parseInt(tokens[tokens.length - 1].token_id), 0, amount * UINT256_DECIMALS, 0]`